removeVowels/removeVowels.js

```diff
@@ -0,0 +1,17 @@
+
+function removeVowels(str){
+
+	let strSplitted = str.split('');
```

Comment:
# Code Review (Nitpick) 
Please fix indentation on this line and make it consistent with the other lines in the function.